src/js/background/background.js

```diff
@@ -46,23 +46,25 @@ chrome.tabs.onRemoved.addListener((tabId) => {
 
 chrome.runtime.onMessage.addListener(
   (request, sender, sendResponse) => {
-    const pageDataObj = {
-      fragments: request.fragmentsData,
-      id: sender.tab.id,
-      url: sender.tab.url,
-    };
+    if (sender.tab) {
```

Comment:
Do we have any tests for background script?

Reply:
currently no.